content/articles/matlab-graphical-user-interface/index.md

```diff
@@ -0,0 +1,107 @@
+### GRAPHICAL USER INTERFACE IN MATLAB
+### Introduction
+A graphical user interface is a pictorial interface that allows the user to use an application without understanding the language.
+This is done by providing intuitive controls. The controls are the buttons that the user clicks to obtain a determined output and we will discuss later them in this article. GUI is an event-driven program. This is because it acquires input at any given time and uses the callback functions to execute the program and give results.
+In this article, we will be looking at the components that are used to construct the graphical user interface. We will also look at how to write the callback function of the components that make it possible for them to execute the controls. Matlab generates the codes for a GUI. This makes it a better tool for designing the GUI for engineering components.
+### Prerequisites
+1. [Matlab](https://www.mathworks.com/downloads/) installed.
+2. A proper understanding of [Matlab](https://www.section.io/engineering-education/getting-started-with-matlab/) language.
+
```

Comment:
If possible, add a `Table of contents` section. It would help readers navigate between topics.